backend/core/views/confirmation.hbs

```diff
@@ -0,0 +1,7 @@
+<h2>{{t "t.hello"}} {{> user-name}},</h2>
```

Comment:
It seems like these views should probably be nested a level deeper - perhaps `views/confirmation_email/*`?

Reply:
or `email_views/confirmation/...`, as long as that doesn't break the expected nestjs pattern